src/main/java/cn/nukkit/player/Player.java

```diff
@@ -4447,6 +4448,27 @@ public void showXboxProfile(String xuid) {
         this.sendPacket(packet);
     }
 
+    /**
+     * Sends a sign change. Sends "fake" sign contents
+     * by sending a sign change packet. This does not actually
+     * modify the sign lines and is only temporary.
+     *
+     * @param position the position of the sign
+     * @param lines the lines to send
+     */
+    public void sendSignChange(Vector3i position, String[] lines) {
```

Comment:
Isn't it better to use a sign block instead of a vector?

Reply:
This isn't related to the block. It's just a way for users to update a sign's text without diving into the protocol.